test/run

```diff
@@ -1296,8 +1320,8 @@ compile() {
 # that the output of bin/compile will not be captured
 compile_and_report() {
   default_process_types_cleanup
-  cache_dir=$(mktmpdir)
-  env_dir=$(mktmpdir)
+  cache_dir=${2:-$(mktmpdir)}
```

Comment:
It's important for the test that the cache directory used in `bin/compile` is re-used for `bin/report`, but this was using different directories

Reply:
It mimics what actually happens during the build, and here we are pulling metadata out of the cache. All of the `meta_get` calls [reference a datastore stored in the cache directory](https://github.com/heroku/heroku-buildpack-nodejs/blob/master/lib/metadata.sh#L10)

Reply:
Gotcha.. makes sense to me 👍